bothub/api/v2/repository/views.py

```diff
@@ -789,6 +827,47 @@ def check_can_automatic_evaluate(self, request, **kwargs):
         return Response(response)  # pragma: no cover
 
 
+    @action(
```

Comment:
this line contains two blank lines, by default we use 1 line for methods and 2 for classes.